share/locale/en.json

```diff
@@ -1,165 +1,165 @@
     "gui_settings_connection_type_socket_file_option": "Connect using socket file",
-    "gui_settings_connection_type_test_button": "Test Tor Settings",
+    "gui_settings_connection_type_test_button": "Test Tor Connection",
     "gui_settings_control_port_label": "Control port",
     "gui_settings_socket_file_label": "Socket file",
     "gui_settings_socks_label": "SOCKS port",
     "gui_settings_authenticate_label": "Tor authentication options",
-    "gui_settings_authenticate_no_auth_option": "No authentication, or cookie authentication",
+    "gui_settings_authenticate_no_auth_option": "None or cookie-based",
     "gui_settings_authenticate_password_option": "Password",
     "gui_settings_authenticate_cookie_option": "Cookie",
     "gui_settings_password_label": "Password",
-    "gui_settings_cookie_label": "Cookie path",
-    "gui_settings_tor_bridges": "Tor Bridge support",
+    "gui_settings_cookie_label": "Cookie location",
+    "gui_settings_tor_bridges": "Tor bridge support",
     "gui_settings_tor_bridges_no_bridges_radio_option": "Don't use bridges",
     "gui_settings_tor_bridges_obfs4_radio_option": "Use built-in obfs4 pluggable transports",
     "gui_settings_tor_bridges_obfs4_radio_option_no_obfs4proxy": "Use built-in obfs4 pluggable transports (requires obfs4proxy)",
     "gui_settings_tor_bridges_meek_lite_azure_radio_option": "Use built-in meek_lite (Azure) pluggable transports",
     "gui_settings_tor_bridges_meek_lite_azure_radio_option_no_obfs4proxy": "Use built-in meek_lite (Azure) pluggable transports (requires obfs4proxy)",
-    "gui_settings_meek_lite_expensive_warning": "Warning: the meek_lite bridges are very costly for the Tor Project to run!<br><br>You should only use meek_lite bridges if you are having trouble connecting to Tor directly, via obfs4 transports or other normal bridges.",
+    "gui_settings_meek_lite_expensive_warning": "Warning: The meek_lite bridges are very costly for the Tor Project to run.<br><br>Only use them if unable to connect to Tor directly, via obfs4 transports or other normal bridges.",
     "gui_settings_tor_bridges_custom_radio_option": "Use custom bridges",
     "gui_settings_tor_bridges_custom_label": "You can get bridges from <a href=\"https://bridges.torproject.org/options\">https://bridges.torproject.org</a>",
-    "gui_settings_tor_bridges_invalid": "None of the bridges you supplied seem to work.\nPlease try again by double-checking them or adding other ones.",
+    "gui_settings_tor_bridges_invalid": "None of your bridges work.\nDouble-checking them or add others.",
     "gui_settings_button_save": "Save",
     "gui_settings_button_cancel": "Cancel",
     "gui_settings_button_help": "Help",
-    "gui_settings_shutdown_timeout_checkbox": "Use auto-stop timer",
-    "gui_settings_shutdown_timeout": "Stop the share at:",
-    "settings_saved": "Settings saved to {}",
+    "gui_settings_shutdown_timeout_checkbox": "Use expiry timer",
```

Comment:
I think "auto-stop" is more clear than "expiry"

Reply:
Well, expiry of what, and the same is true for my alteration. To revise,

"Stop OnionSharing after a given time." and
"Set timeframe for OnionSharing expired"?